cmd/stackdriver-prometheus-sidecar/main.go

```diff
@@ -231,7 +232,10 @@ func main() {
 	a.Flag("web.listen-address", "Address to listen on for UI, API, and telemetry.").
 		Default("0.0.0.0:9091").StringVar(&cfg.listenAddress)
 
-	a.Flag("filter", "PromQL-style label matcher which must pass for a series to be forwarded to Stackdriver. May be repeated.").
+	a.Flag("filterset", "PromQL metric and label matcher which must pass for a series to be forwarded to Stackdriver. If repeated, the series must pass any filter set to be forwarded.").
```

Comment:
I felt that filterset does not serve well for the value we accept - as the value of filterset is not just comma split string (which fits the name `filterset` more), but instant vector selector from PromQL.

Also `filterset` and `filter` are really similar, which may confuse our users. Can you find an alternative name here?

Reply:
I agree that `filterset` is not a great name, though being related to `filter` makes it clear that both flags are controlling the same thing.

Do you like `--whitelist` or `--metric` better?

Reply:
Thanks for coming up with the name. Whitelist sounds good for me.